yjit.c

```diff
@@ -1169,6 +1169,7 @@ rb_yjit_set_exception_return(rb_control_frame_t *cfp, void *leave_exit, void *le
 
 // Primitives used by yjit.rb
 VALUE rb_yjit_stats_enabled_p(rb_execution_context_t *ec, VALUE self);
+VALUE rb_yjit_verbose_stats_p(rb_execution_context_t *ec, VALUE self);
```

Comment:
Very much a nitpick but could you rename it to `print_stats_p`. Verbose kind of implies that there is a non-verbose mode. I think this would be more descriptive. Everything else looks good 👌